<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QListView" name="images_listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>191</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="char_graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="result_graphicsView">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_lineEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>480</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>700</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
